### DISCUSSION TITLE
README incorrectly implies that age-plugin-yubikey broadly supports PIV tokens

### DISCUSSION THREAD

**Discussion by RespiteSage** [FiloSottile/age#592] [General]
The `age` README file says that

> Hardware PIV tokens such as YubiKeys are supported through the [age-plugin-yubikey](https://github.com/str4d/age-plugin-yubikey) plugin.

but `age-plugin-yubikey` only supports a particular subset of PIV tokens (per [this issue](https://github.com/str4d/age-plugin-yubikey/issues/62#issuecomment-1367915274), P256 tokens in the 20 "retired" slots).

This isn't a huge issue, but clarifying the language might save someone else the time it took me to try out `age-plugin-yubikey` and look through issues before I found the issue linked above.

I think something like

> Some PIV tokens, including YubiKeys, are supported through the [age-plugin-yubikey](https://github.com/str4d/age-plugin-yubikey) plugin.

would be enough of a change.

**Comment by str4d** [comment]
I replied to the duplicate of this in my repo: https://github.com/str4d/rage/issues/541#issuecomment-2418574369

tl;dr "PIV token" means the hardware device, not the key material stored on it.

**Reply by RespiteSage** [reply]
I'm continuing the discussion in the `rage` issue. If our discussion reaches a conclusion there, I'll add a summary here in a comment.
